pkg/types/types.go

```diff
@@ -37,6 +37,7 @@ type ResourceConfig struct {
 		Vendors []string `json:"vendors,omitempty"`
 		Devices []string `json:"devices,omitempty"`
 		Drivers []string `json:"drivers,omitempty"`
+		NetDevs []string `json:"ifnames,omitempty"`
```

Comment:
The filed name `NetDevs` is ambiguous. if its not too much to ask, lets make this field name(and related methods) obvious to what this selector is doing. I think "pfNames" goes well with this selector. What do you think? (I know the original PR was intended for actual NetDev name for VFs and you're probably reusing the var names etc. Lets try to keep the identifiers meaningful so less comments/docs are needed to explain them).

Reply:
fixed